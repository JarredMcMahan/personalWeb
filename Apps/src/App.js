import React from 'react';
import './style/App.css';
import "bootstrap/dist/css/bootstrap.min.css";
import Navbar from "./components/Navbar";
import Front from './components/Front';
import Legal from './components/Legal';

function App() {
  return (
    <dev>
      <Navbar />
      <Front />
      <Legal />
    </dev>
  );
}

export default App;
