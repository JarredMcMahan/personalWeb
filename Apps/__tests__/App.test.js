import React from 'react';
import { render, cleanup } from '@testing-library/react';
import App from '../src/App';
import '@testing-library/jest-dom';

afterEach(cleanup)

it('should take a snapshot', () => {
  const { asFragment } = render(<App />)
  expect(asFragment(<App />)).toMatchSnapshot()
});
