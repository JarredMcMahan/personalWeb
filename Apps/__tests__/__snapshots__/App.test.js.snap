// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <nav
      class="navbar navbar-expand-lg navbar-dark bg-dark"
    >
      <a
        class="navbar-icon "
        href="Logo"
      >
        <img
          alt="Logo"
          src="[object Object]"
        />
      </a>
      <button
        aria-controls="navbarSupportedContent"
        aria-expanded="false"
        aria-label="Toggle navigation"
        class="navbar-toggler"
        data-target="#navbarSupportedContent"
        data-toggle="collapse"
        type="button"
      >
        <span
          class="navbar-toggler-icon"
        />
      </button>
      <div
        class="collapse navbar-collapse"
        id="navbarSupportedContent"
      >
        <ul
          class="navbar-nav ml-auto"
        >
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="Home"
            >
              Home 
              <span
                class="sr-only"
              >
                (current)
              </span>
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="Projects"
            >
              Projects
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="About"
            >
              About me
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="nav-link"
              href="Contact"
            >
              Contact
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </div>
</DocumentFragment>
`;
